xtask/src/release.rs

```diff
@@ -78,8 +158,18 @@ impl flags::Release {
         Ok(changes.trim().to_owned())
     }
 
+    /// Get the name of this release.
+    fn get_name(&self) -> String {
```

Comment:
```suggestion
    fn title(&self) -> String {
```

Or something like that.